dowhy/gcm/util/general.py

```diff
@@ -185,6 +185,15 @@ def means_difference(randomized_predictions: np.ndarray, baseline_values: np.nda
     return np.mean(randomized_predictions).squeeze() - np.mean(baseline_values).squeeze()
 
 
+def variance_of_deviations(randomized_predictions: np.ndarray, baseline_values: np.ndarray) -> np.ndarray:
+    return -np.var((randomized_predictions - baseline_values).squeeze())
```

Comment:
why is there a negative sign for `np.var`?

Reply:
Since the Shapley values estimate the quantity ```v(S u {i}) - v(S)``` for a subset ```S```. In case of variance, the variance becomes larger if we take features out, i.e. ```v(S u {i}) <= v(S)```. Therefore, without the minus, we would obtain a negative contribution of a feature to the variance, which is rather confusing. This is typically the case with all the uncertainty measures, where the uncertainty increases when we remove a feature.